java/ql/lib/semmle/code/java/dataflow/FlowSummary.qll

```diff
@@ -105,15 +105,15 @@ private module SyntheticCallables {
   private import semmle.code.java.frameworks.Stream
 }
 
-private newtype TSummarizedCallableBase =
+private newtype TModelingCandidateCallable =
   TSimpleCallable(Callable c) { c.isSourceDeclaration() } or
   TSyntheticCallable(SyntheticCallable c)
 
 /**
- * A callable that may have a flow summary. This is either a regular `Callable`
+ * A callable that may have a model. This is either a regular `Callable`
  * or a `SyntheticCallable`.
  */
-class SummarizedCallableBase extends TSummarizedCallableBase {
+class ModelingCandidateCallable extends TModelingCandidateCallable {
```

Comment:
By inspecting this again, I think it needs to re-written. The claims I have made are too strong (especially because there is currently no connection between sources and sinks and then this class - however I think it might be doable to make, but it is out of scope for this PR). Furthermore, since java has synthetic callables, we probably need to separate base classes for summary and neutral candidates.
I will need to do some more work on this.

I don't think you should base the `ModelApi` on any of the classes already related to modelling from the `FlowSummary` like files as we using it to test these classes.